apartmenttherapy/order-service
# Elaine/order by id

#### What does this PR do & why?
I believe this does a couple things:
- adds an abstracted method that will retrieve an order and hydrate its address;
- has the `create` endpoint return a hydrated order.

#### How can this be manually tested?
- Tests should run.
- Both `create` and `update` mutations should return orders with hydrated addresses.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1110870397370745

#### GIF


#### Have you remembered to...?
